datastore/package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.0.1",
   "private": true,
   "license": "Apache Version 2.0",
+  "bin": {
```

Comment:
Well, to follow how authSamples does it, it will be `npm run getting-started.js -- list`. It's not the prettiest way to execute, but it does help us avoid other potential issues.